Backend/src/server.ts

```diff
@@ -10,24 +10,52 @@ import { ApiError } from "./customErrors/apiError";
 import initPassport from "./middlewares/passportSetup";
 import passport from "passport";
 import { requestLogger, errorLogger } from "./middlewares/logger";
+import BruteForceDetector from "./util/bruteForceDetector";
+import eventEmitter from "./util/CustomEmitter";
 const jwt = require("jsonwebtoken");
 const bodyParser = require("body-parser");
 const app = express();
 const path = require("path");
 require("dotenv").config({ path: path.join(process.cwd(), ".env") });
 
+const BFD = new BruteForceDetector(1000);
+// milliseconds
+const banTime = 5000;
+// List of temporarily banned IP addresses
+const addressList = new Map();
+
 initPassport();
 
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(bodyParser.json());
 
 app.use(passport.initialize());
 
-app.use(requestLogger);
 //The regular logger needs to be before the router
-app.use(requestLogger);
+// app.use(requestLogger);
```

Comment:
Bliver den slet ikke brugt nu så?

Reply:
I only removed it because I was testing something else - forgot to remove the comment. We're still supposed to use it